### ISSUE TITLE
API Explorer shows Petstore instead of custom API

### ISSUE THREAD

**Issue by nerdDan** [BiteBit/koa-oai-router#43]
I configured the router according to document.

```typescript
const router = new Router({
  apiDoc: path.join(__dirname, "api"),
  apiExplorerVisible: true
});
router.mount(middleware, path.join(__dirname, "controllers"));
```

All the APIs (`http://localhost:8080/api/...`) works fine.

The problem is, the API Explorer shows Petstore instead of my API.

![deepinscreenshot_select-area_20180623022713](https://user-images.githubusercontent.com/3108734/41792920-0fd80f52-768d-11e8-8238-796ba808f156.png)

**Comment by nerdDan** [comment]
It works when I change the `https://petstore.swagger.io/v2/swagger.json` into `api.json` in that `<input>`.
But I guess `api.json` should be a default value.

**Comment by BiteBit** [comment]
I'm sorry for this bug because the `swagger-ui-dist` is updated.

I try to replace the `http://petstore.swagger.io/v2/swagger.json` with `api-explorer-config.json`.

https://github.com/BiteBit/koa-oai-router/blob/575486e40a0ed1eaf7af22035105934c4e5ae5c7/src/swagger.js#L18-L20

`index.html` in `swagger-ui-dist` is updated to `https://petstore.swagger.io/v2/swagger.json`.
 So the replacement is failed.

I will fix it right now.

**Comment by BiteBit** [comment]
Fixed, please update to `2.0.2`.
